aerofiles/seeyou/reader.py

```diff
@@ -193,3 +193,90 @@ def decode_description(self, description):
             return None
 
         return description
+
+    def decode_taskname(self, fields):
+        if not fields:
+            raise ParserError('Name field must not be empty')
+
+        return fields[0]
+
+    def decode_tasklist(self, fields, waypoints):
```

Comment:
i'm not sure if i follow you correctly as the implementation of a reader method was really straight forward. i have run it with a local main file and results seem the same as the original

```
def read(self, fp):
        waypoints = filter(None, map(self.decode_waypoint, csv.reader(fp)))
        return dict(waypoints=waypoints)
```